src/helpers/refreshIntervalHelper.js

```diff
@@ -52,6 +52,10 @@ export const refreshTimeFor = async (refreshTimeKey, refreshInterval) => {
 
       break;
     }
+    case consts.REFRESH_INTERVALS.NEVER: {
+      // always return tomorrow, so that we never refresh
+      return moment().add(1, 'days').unix();
+    }
```

Comment:
do we need to delete possibly existing refresh time from AsyncStorage in this case? it could be, that there is something saved from the past, right, through the `ONCE_A_DAY` case here that was used before?

Reply:
i don't think we need to delete those refresh times. we do not clean up previously stored refresh times for any other "switching" of refresh times either. in the case of `NEVER`, we simply ignore the previously stored refresh time.

Reply:
right, we have no benefit of including cleanup logics